kafka/pykafka/connection.py

```diff
@@ -30,7 +30,8 @@
 class BrokerConnection(object):
     """A socket connection to Kafka."""
 
-    def __init__(self, host, port):
+    def __init__(self, host, port, buffer_size=64 * 1024):
+        self._buff = bytearray(buffer_size)
```

Comment:
This is a public repo, and this deserves public applause. Again, really good catch that we can eliminate a lot of memory allocations by simply re-using this buffer.  :clap: :clap: :clap: